docs/tips/14-communicate-between-components.md

```diff
@@ -10,30 +10,40 @@ next: expose-component-functions.html
 For parent-child communication, simply [pass props](/react/docs/multiple-components.html).
 
 For child-parent communication:
-Say your `GroceryList` component has a list of items generated through an array. When a list item is clicked, you want to display its name:
+Say your `GroceryList` component has a list of `Item`s generated through an array. When a list item is clicked, you want to display its name:
 
 ```js
-var handleClick = function(i, props) {
-  console.log('You clicked: ' + props.items[i]);
-}
+var GroceryList = React.createClass({
+  show: function(e) {
+    console.log(e);
+  },
+  render: function() {
+    return (
+      <div>
+        {this.props.items.map((item, index) => {
+          return <Item key={index} value={item} log={this.show}/>
+        })}
+      </div>
+    );
+  }
+})
 
-function GroceryList(props) {  
-  return (
-    <div>
-      {props.items.map(function(item, i) {
-        return (
-          <div onClick={handleClick.bind(this, i, props)} key={i}>{item}</div>
-        );
-      })}
-    </div>
-  );
-}
+var Item = React.createClass({
+  handleClick: function(e) {
+    this.props.log(this.props.value);
+  },
+  render: function() {
+    return (
+      <div onClick={this.handleClick}>{this.props.value}</div>
+    );
+  }
+})
 
 ReactDOM.render(
   <GroceryList items={['Apple', 'Banana', 'Cranberry']} />, mountNode
 );
 ```
 
-Notice the use of `bind(this, arg1, arg2, ...)`: we're simply passing more arguments to `handleClick`. This is not a new React concept; it's just JavaScript.
+Notice the use of the arrow function `=>`: we're ensuring `this` inside the `map()` refers to the `GroceryList`. This is not a new React concept; it's just JavaScript.
```

Comment:
Sounds good, just a couple additions to make it flow a little better (but not actually emphasized).

> `this` inside the `map()` **_call_** refers to the `GroceryList` **_component_**.